specification/sql/resource-manager/Microsoft.Sql/preview/2019-06-01-preview/examples/ManagedDatabaseRestoreDetails.json

```diff
@@ -0,0 +1,33 @@
+{
+  "parameters": {
+    "subscriptionId": "00000000-1111-2222-3333-444444444444",
+    "resourceGroupName": "Default-SQL-SouthEastAsia",
+    "managedInstanceName": "managedInstance",
+    "databaseName": "testdb",
+    "restoreDetailsName": "Default",
+    "api-version": "2018-06-01-preview",
```

Comment:
Should this be 2019-06-01 instead, or it is intentionally 2018 for backward compatibility?